test/hotspot/jtreg/compiler/jvmci/compilerToVM/DisassembleCodeBlobTest.java

```diff
@@ -101,7 +101,16 @@ private void check(CompileCodeTestCase testCase) {
         // Therefore compare strings 2 and 3.
         String str2 = CompilerToVMHelper.disassembleCodeBlob(installedCode);
         String str3 = CompilerToVMHelper.disassembleCodeBlob(installedCode);
-        Asserts.assertEQ(str2, str3,
+        String[] str2Lines = str2.split(System.lineSeparator());
+        String[] str3Lines = str3.split(System.lineSeparator());
+        // skip the first two lines since it contains a timestamp that may vary from different invocations
+        // <empty-line>
+        // Compiled method (c2)     309  463       4       compiler.jvmci.compilerToVM.CompileCodeTestCase$Dummy::staticMethod (1 bytes)
+        // <empty-line>
+        // Compiled method (c2)     310  463       4       compiler.jvmci.compilerToVM.CompileCodeTestCase$Dummy::staticMethod (1 bytes)
+        for (int i = 2; i < str2Lines.length; i++) {
+            Asserts.assertEQ(str2Lines[i], str3Lines[i],
```

Comment:
Splitting the entire string by lines seems like a bit of an overhead. What about something like that (not tested)?
```
int idx = str2.indexOf(System.lineSeparator());
idx = str2.indexOf(System.lineSeparator(), idx + 1);
str2 = str2.substring(idx + 1);
```

Reply:
Doing it by lines will actually provide a more focused error message if there's a problem and I cannot image the overhead matters for a test like this.

Reply:
Right, good point.